src/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,35 @@
+import styles from './Navigation.module.scss'
+function Navigation(): JSX.Element {
+    return(
+        <>
+        <div className={styles.Navigation}>
+            <div className={styles.leftContent}>
+                <div className={styles.BeehiveLogo}>
+                    <img className={styles.logo} src="public/Subtract.png" alt='beehive logo'/>
+                    <span className={styles.name}>Name</span>
+                </div>
+                <ul className={styles.navbarMenu}>
+                    <li className={styles.navbarMenuItem}>Portfolio</li>
```

Comment:
## Stretch Goal
_not required_
- Turn these li tags into their own `NavBarMenuItem` component. This way they could hold state like a `isHilighted` property. When `isHighlighted` is `true` then the menu item is highlighted blue. 
	- This would better prepare us for when we eventually start adding navigation functionality to the site. The user would be able to see which page they are currently on because it would be highlighted blue.

Reply:
This sure is a stretch goal, let's see if Pai and I can figure out how to do this